Addendum

ElDeveloper opened the pull request:
I've added the HTML formatted version of the Addendum as authored by @JWDebelius. This is likely not ready to be merged. I've made it available via the [following page](http://webdev.microbio.me/americangut/addendum.psp), to get to this page users will have to login with their account and click in the left sidebar "The American Gut Addendum", which should open a new page with all the information that's currently contained in this document.

@JWDebelius, would you mind taking a look at it and making sure everything is looking good?

ElDeveloper committed:
Add link to the addendum page

ElDeveloper committed:
Add addendum.psp page as created by @JWDebelius

This page was created by @JWDebelius but I formatted it and integrated
it with the current CSS and website.

ElDeveloper committed:
Add resources for addendum.psp

These resources were gathered by @JWDebelius.

jwdebelius committed:
Added a linked certificate

Clicking on plots on the certificate will now take the particiapnt to
the description.
Formatting was also changed so tables and h6 look like text from the
certificate.

teravest commented on www/americangut/addendum.psp:
```diff
@@ -0,0 +1,339 @@
+                    <div class="faq">
+                        <div id = 'text' align="justify">
+                            <a name = "top"><h1>American Gut Addendum</h1></a>
+                            <p>First, we&apos;d like to note that in general these data allow you to understand how similar or different you are to other people in terms of the bacterial composition of the sample you sent. The information about the microbes is at as fine level of a taxonomic resolution as we were able to achieve with our sequencing methods, and varies for different groups of micr ...
```
Take out First we'd like to note that in General.  start with These data...

teravest commented on www/americangut/addendum.psp:
```diff
@@ -0,0 +1,339 @@
+                                <li>Order: Primates</li>
+                                <li>Family: Hominidae</li>
+                                <li>Genus: <em>Pan</em></li>
+                                <li>Species: <em>troglodyte</em></li>
```
should be troglodytes

teravest commented on www/americangut/addendum.psp:
```diff
@@ -0,0 +1,339 @@
+                            </tr>
+                        </table>
+
+                        <p> The second table shows four microbes that you had more of compared to other people. It is likely that other participants also have these microbes in their sample, but we found substantially higher abundances of them in your sample relative to others. The percentages on the right tell you how many of your total bacteria (under &quot;Sample&quot;) or of the total bacteri ...
```
this has "person&s" instead of "person's"

teravest commented on www/americangut/addendum.psp:
```diff
@@ -0,0 +1,339 @@
+
+                        <h3>Rare Taxa</h3>
+
+                        <h6>This sample included the following rare taxa: Fenus <em>Varibaculum</em>, Genus <em>Neisseria</em>, Genus <em>Campylobacter</em>, Unclassified Order ML615J-28.</h6>
```
the font on this line is way too small

teravest commented on www/americangut/addendum.psp:
```diff
@@ -0,0 +1,339 @@
+                        <h3>Different Body Sites</h3>
+                        <img src = "img/PCoA1.png"
+                             alt = "PCoA by body site for AGP and HMP"/>
+                        <p>This plot lets you compare your sample to samples collected by two microbiome projects from several body sites. The color of each point tells you which project and body site the sample came from. HMP refers to the <a href = 'http://www.hmpdacc.org'>Human Microbiome Project</a>, funded by the National Institutes of Health. You can see how your sample compared to fecal, o ...
```
should be other microbiome projects instead of two microbiome progjects

teravest commented on www/americangut/addendum.psp:
```diff
@@ -0,0 +1,339 @@
+                        <p>A relatively new phylum with only a handful of described species. Although not the most abundant, they seem to be always present in soil, aquatic environments, and feces. Most likely they are involved in the decomposition of organic matter, with no known pathogens. While some may be autotrophs, others can be internal symbionts of microscopic eukaryotes such as protists  ...
+
+                        <h4>Tenericutes</h4>
+                        <p>A phylum of Gram-negative bacteria without a cell wall (<em>tener</em> - soft, <em>cutis</em> - skin) which are Organized in a single class. Nutritionally, they represent variable pathways ranging from aerobic and anaerobic fermenters to commensals to strict pathogens of vertebrates (e.g., fish, cattle, wildlife). Among the best studied are Mycoplasmas with a fried egg- ...
```
Organized shouldn't be capatilzed

jwdebelius committed:
Corrects for gramatical changes

ElDeveloper commented:
@JWDebelius are all corrections done now?

jwdebelius commented:
No, I still have to implement the suggestions Meg and Dorota made.

On Tue, Dec 10, 2013 at 10:39 AM, Yoshiki Vázquez Baeza <
notifications@github.com> wrote:

> @JWDebelius https://github.com/JWDebelius are all corrections done now?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/qiime/qiime_web_app/pull/545#issuecomment-30249710
> .

ElDeveloper commented:
Cool. Let me know.

Thanks!

Yoshiki.

On Dec 10, 2013, at 11:51 AM, J W Debelius notifications@github.com wrote:

> No, I still have to implement the suggestions Meg and Dorota made. 
> 
> On Tue, Dec 10, 2013 at 10:39 AM, Yoshiki Vázquez Baeza < 
> notifications@github.com> wrote: 
> 
> > @JWDebelius https://github.com/JWDebelius are all corrections done now? 
> > 
> > — 
> > Reply to this email directly or view it on GitHubhttps://github.com/qiime/qiime_web_app/pull/545#issuecomment-30249710 
> > . 
> > 
> > —
> > Reply to this email directly or view it on GitHub.

jwdebelius committed:
Certificate includes css transparancies

Certificate at the top of the page appears with transparancy (hover
over for full color) in Safari. Firefox link version not yet working.

ElDeveloper commented:
Awesome, thank @JWDebelius this is looking freaking amazing!

jwdebelius committed:
Made opacity work in Firefox/IE

wasade commented:
Can this be merged?

jwdebelius commented:
I have not heard anything new from anyone, so I would say yes.

On Thu, Dec 26, 2013 at 11:25 AM, Daniel McDonald
notifications@github.comwrote:

> Can this be merged?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/qiime/qiime_web_app/pull/545#issuecomment-31230252
> .
